weave-js/src/components/PagePanelComponents/Home/Browse3.tsx

```diff
@@ -333,35 +337,39 @@ const MainPeekingLayout: FC = () => {
           flex: '1 1 40%',
           overflow: 'hidden',
           display: 'flex',
+          // This transition is from the mui drawer component, to keep the main content animation in similar
+          transition: !isDrawerOpen
+            ? 'margin 225ms cubic-bezier(0, 0, 0.2, 1) 0ms'
+            : 'none',
+          marginRight:
+            !isDrawerOpen || !isFlexRow
+              ? 0
+              : // this is a hack to make the drawer width percentage work with the siderbar of 56px
+                `${drawerWidthPct / (1 - 56 / windowSize.width)}%`,
+          // this is vh because margin-bottom percentages are relative to the viewport width
+          marginBottom: !isDrawerOpen || isFlexRow ? 0 : `${drawerHeightPct}vh`,
```

Comment:
curious why we need to account for the 56 pixels in the sidebar, but not the header bar

Reply:
me too, its really weird, i couldnt figure it out